app/change_set_persisters/change_set_persister.rb

```diff
@@ -103,11 +103,12 @@ def buffer_into_index
         yield self
         return
       end
+
+      delayed_queue = nil
       metadata_adapter.persister.buffer_into_index do |buffered_adapter|
         with(metadata_adapter: buffered_adapter) do |buffered_changeset_persister|
           yield(buffered_changeset_persister)
-          @created_file_sets = buffered_changeset_persister.created_file_sets
```

Comment:
Are we dependent on the `@created_file_sets` variable being set when the callbacks are run, e.g., `after_commit`: https://github.com/pulibrary/figgy/pull/2191/files#diff-f98978ff0ab2af320d010f45a93af6ceR214

Reply:
I thought we were, but all the tests pass.

Reply:
Upload works in development too. I think because the handlers get the created file sets passed to them.